e2e/fixtures/use-router/src/pages/dynamic.tsx

```diff
@@ -1,14 +1,22 @@
-import { Link } from 'waku';
+'use client';
```

Comment:
This `/dynamic` page is used for other specs, so it feels safe to avoid changing it to a client component.